src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -15,10 +16,23 @@ const useStyles = makeStyles((theme) => ({
     height: theme.spacing(9),
     margin: "12px auto 8px auto",
   },
+  profileEditButton: {
+    backgroundColor: "transparent",
+    color: theme.color.darkBlue,
+    "&:hover": {
+      backgroundColor: "transparent",
+      color: theme.color.secondaryBlue,
+    },
+  },
 }));
 
 const UserStatus = ({ profile, setProfile, view }) => {
   const classes = useStyles();
+  const [loaded, error] = useScript(
+    `https://maps.googleapis.com/maps/api/js?key=${process.env.REACT_APP_MAPS_ACCESS_TOKEN}&libraries=places`
```

Comment:
Do we have any documentation updates on where this new environment variable is coming from? If we're merging this in, this feature will not work if we're expecting another local environment variable.

Reply:
Addressed in 0d3f857. We should update the example .env file that is pinned to the slack channel to include `REACT_APP_MAPS_ACCESS_TOKEN`